data/data/azure/vnet/variables.tf

```diff
@@ -55,3 +51,24 @@ variable "private_dns_zone_id" {
   type        = string
   description = "This is to create explicit dependency on private zone to exist before VMs are created in the vnet. https://github.com/MicrosoftDocs/azure-docs/issues/13728"
 }
+
+variable "preexisting_network" {
+  type    = bool
+  default = false
+}
+
+variable "virtual_network" {
+  type = string
```

Comment:
Can you add some descriptions here maybe?

Reply:
I've added descriptions!